test/.vscode/launch.json

```diff
@@ -0,0 +1,30 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
+// Use IntelliSense to learn about possible attributes.
+// Hover to view descriptions of existing attributes.
+// For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+{
```

Comment:
Does the `.vscode` directory do anything if it's not in the repository root?  At least I couldn't find the launch configurations in vscode.

Reply:
it works for me if I cd into the "test" folder and run "code .", to run the unit tests I'm also doing "cd test" then "npm run test"

Reply:
this is mostly just a clone of [helloworld-test-sample](https://github.com/microsoft/vscode-extension-samples/tree/main/helloworld-test-sample) from [working-with-extensions/testing-extension](https://code.visualstudio.com/api/working-with-extensions/testing-extension).

Reply:
It's a little awkward to have to open the `test/` directory as its own workspace. Why not put the launch configs in the top-level `.vscode` configuration?

Reply:
Sounds resolved, thanks!

Reply:
Well not quite, the debugger doesn't seem to launch the test correctly, the test runs, but the setup in runtests.ts is not working, I get weird errors like this:
```
[d:\git\leanprover\vscode-lean4]: property `version` is mandatory and must be of type `string`
```
and the runtest should be opening the `src\lean\lean3` folder but it is failing to do so... so I could use some help debugging this one...

Reply:
this is now resolved and tests are passing and are debuggable.